python/podio_class_generator.py

```diff
@@ -543,6 +535,88 @@ def create_collection(self, classname, definition):
       self.fill_templates("Collection",substitutions)
       self.created_classes.append("%sCollection"%classname)
 
+    def create_PrintInfo(self, classname, definition):
+        namespace, rawclassname, namespace_open, namespace_close = self.demangle_classname(classname)
+
+        toFormattingStrings = {"char" : "3" , "unsigned char" : "3", "long":"11", "longlong":"22", "bool":"1", "int":""}
+        formats = {"char" : 3 , "unsigned char" : 3, "long": 11, "longlong": 22 , "bool": 1,}
+
+        outputSingleObject = 'o << "' + classname + ':" << std::endl; \n     o << '
+        WidthIntegers = ""
+        widthOthers = ""
+        findMaximum = ""
+        setFormats = ""
+        formattedOutput = ""
+        tableHeader = ""
+        for member in definition["Members"]:
+            name = member["name"]
+            lengthName = len(name)
+            klass = member["type"]
+            if(klass in toFormattingStrings  and not klass == "int"):
+                if(formats[klass] > lengthName):
+                    widthOthers = widthOthers + "  int"+ " " + name + "Width = " + toFormattingStrings[klass] + " ; \n"
+                else:
+                    widthOthers = widthOthers + "  int" + " " + name + "Width = " + str(lengthName) + " ; \n"
+            elif klass == "int":
```

Comment:
this and the other code further below you may want to put into https://github.com/hegner/podio/blob/master/python/podio_templates.py